content/docs/command-reference/remote/modify.md

```diff
@@ -108,6 +108,20 @@ these settings, you could use the following options:
   $ dvc remote modify myremote endpointurl https://myendpoint.com
   ```
 
+- `access_key_id` - AWS Access Key ID. Optional, overrides credentials from
+  `credentialpath`:
+
+  ```dvc
+  $ dvc remote modify myremote access_key_id my-access-key-id
+  ```
+
+- `secret_access_key` - AWS Secret Access Key. Optional, Overrides credentials
```

Comment:
I'll commit this:

```suggestion
- `access_key_id` - (optional) AWS Access Key ID. Overrides credentials from
  `credentialpath`:

  ```dvc
  $ dvc remote modify myremote access_key_id my-access-key-id
  ```

- `secret_access_key` - (optional) AWS Secret Access Key. Overrides credentials
```